brave/browser-laptop
# Implemented notification for user to backup Brave wallet

Resolves #13425
Unblocks #4847 

## Submitter Checklist:


## Test Plan:

- Start Brave in staging environment and short interval timers from a console: 
`LEDGER_ENVIRONMENT=staging LEDGER_NOTIFICATION=true npm start`
- Keep Note of counter on console
- Enable Payments
- Add funds to wallet
- Counter will be count up to notification show (in ms)
- After 2 minutes, notification will display. Dismiss notification
- After additional 5 minutes (from first notification show), notification will display. Dismiss notification
- Each additional 1 minutes after 2nd notification is shown, notification will display.
- Notification will continue to be displayed 1 once a minute until a backup or recovery action occurs or notifications are opted out.
- Notifications will not display if a backup or recovery action has occurred.

Backup or Recovery action consists of:
Preferences->Payments->Advanced Settings
1. Click 'Backup Your Wallet'. Click 'Copy To Clipboard button'
2. Click 'Backup Your Wallet'. Click 'Print Key'
3. Click 'Backup Your Wallet'. Click 'Save Recovery File...' and save the file to disk
4. Click 'Recover Your Wallet'. Enter a valid recovery key and click 'Recover'
5. Click 'Recover Your Wallet'. Click 'Import Recovery Key'. Import a valid recovery file

unittest -- --grep="showBackupKeys" for simulated

## Reviewer Checklist:


Tests
